From m.kozlowski@tuxland.pl  Tue Feb 17 18:14:52 2009
From: Mariusz Kozlowski <m.kozlowski@tuxland.pl>
Date: Mon, 16 Feb 2009 21:31:29 +0100
Subject: Staging: comedi: usbduxfast: annotate __init and __exit functions
To: greg@kroah.com
Cc: m.kozlowski@tuxland.pl
Message-ID: <1234816289-9859-5-git-send-email-m.kozlowski@tuxland.pl>


This patch adds __init and __exit annotations to proper functions in the
usbduxfast driver.

Signed-off-by: Mariusz Kozlowski <m.kozlowski@tuxland.pl>
Cc: Bernd Porr <BerndPorr@f2s.com>
Cc: David Schleef <ds@schleef.org>
Cc: Frank Mori Hess <fmhess@users.sourceforge.net>
Cc: Ian Abbott <abbotti@mev.co.uk>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/staging/comedi/drivers/usbduxfast.c |    6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

--- a/drivers/staging/comedi/drivers/usbduxfast.c
+++ b/drivers/staging/comedi/drivers/usbduxfast.c
@@ -1826,7 +1826,7 @@ static comedi_driver driver_usbduxfast =
 	.detach		= usbduxfast_detach
 };
 
-static void init_usb_devices(void)
+static void __init init_usb_devices(void)
 {
 	int index;
 
@@ -1875,7 +1875,7 @@ static struct usb_driver usbduxfastsub_d
  * Can't use the nice macro as I have also to initialise the USB subsystem:
  * registering the usb-system _and_ the comedi-driver
  */
-static int init_usbduxfast(void)
+static int __init init_usbduxfast(void)
 {
 	printk(KERN_INFO
 	       KBUILD_MODNAME ": " DRIVER_VERSION ":" DRIVER_DESC "\n");
@@ -1888,7 +1888,7 @@ static int init_usbduxfast(void)
 /*
  * deregistering the comedi driver and the usb-subsystem
  */
-static void exit_usbduxfast(void)
+static void __exit exit_usbduxfast(void)
 {
 	comedi_driver_unregister(&driver_usbduxfast);
 	usb_deregister(&usbduxfastsub_driver);
